doc/source/user/c-info.beyond-basics.rst

```diff
@@ -472,7 +472,7 @@ The __array_finalize\__ method
    members are filled in. Finally, the :obj:`~numpy.class.__array_finalize__`
    attribute is looked-up in the object dictionary. If it is present
    and not None, then it can be either a CObject containing a pointer
-   to a :c:func:`PyArray_FinalizeFunc` or it can be a method taking a
+   to a ``PyArray_FinalizeFunc`` or it can be a method taking a
```

Comment:
I think we should leave this as a hanging reference to remind us to document this function. Unfortunately it is not even documented in the [public header file](https://github.com/numpy/numpy/blob/df6d2600c51502e1877aac563658d0616a75c5e5/numpy/core/include/numpy/ndarraytypes.h#L793) where it is declared, and I don't see any example use of it in tests.